app/services/to_collection_form/mapper.rb

```diff
@@ -42,5 +42,13 @@ def participant_attributes(role)
         { sunetid: participant.sunetid }
       end
     end
+
+    def contact_emails_attributes
+      cocina_description.contact_emails.map { |email| { email: } }
+    end
+
+    def cocina_description
+      @cocina_description ||= CocinaParsers::Description.new(cocina_object:)
```

Comment:
When CocinaParsers::Description is used a bunch, we can instantiate which makes using it a bit cleaner.